token_update_transaction_e2e_test.go

```diff
@@ -843,3 +843,212 @@ func TestIntegrationTokenUpdateTransactionCannotUpdateImmutableNFT(t *testing.T)
 	_, err = resp.SetValidateStatus(true).GetReceipt(env.Client)
 	require.ErrorContains(t, err, "TOKEN_IS_IMMUTABLE")
 }
+
+func TestIntegrationTokenUpdateTransactionUpdateAdminKeyToEmptyKeyList(t *testing.T) {
+	t.Parallel()
+	env := NewIntegrationTestEnv(t)
+
+	resp, err := NewTokenCreateTransaction().
+		SetTokenName("ffff").
+		SetTokenSymbol("F").
+		SetAdminKey(env.OperatorKey).
+		SetSupplyKey(env.OperatorKey).
+		SetTokenType(TokenTypeNonFungibleUnique).
+		SetTreasuryAccountID(env.Client.GetOperatorAccountID()).
+		SetFreezeDefault(false).
+		Execute(env.Client)
+	require.NoError(t, err)
+	receipt, err := resp.SetValidateStatus(true).GetReceipt(env.Client)
+	require.NoError(t, err)
+
+	tx, err := NewTokenUpdateTransaction().
+		SetTokenID(*receipt.TokenID).
+		SetAdminKey(NewKeyList()).
+		SetKeyVerificationMode(NO_VALIDATION).
+		FreezeWith(env.Client)
+
+	// Sign with the current admin key
+	resp, err = tx.Sign(env.OperatorKey).Execute(env.Client)
+	assert.NoError(t, err)
+
+	receipt, err = resp.SetValidateStatus(true).GetReceipt(env.Client)
+	require.NoError(t, err)
+
+}
+
+func TestIntegrationTokenUpdateTransactionUpdateAdminKeyWithoutAlreadySetAdminKeyFails(t *testing.T) {
+	t.Parallel()
+	env := NewIntegrationTestEnv(t)
+
+	zeroAdminKey, err := PrivateKeyFromString("0000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
nit: Maybe you can somehow define the zeroAdminKey string. It looks bad having all that zeros in the tests.

Reply:
added zeroKeyString const